src/docs/introduction/start-customizing/index.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Start customizing your marketplace
+slug: introduction-to-customizing
+updated: 2023-10-25
+category: introduction
+ingress:
+  When you want to start customizing your marketplace, you need to take
+  some steps. Learn what to take into consideration when starting custom
+  development
+published: true
+---
+
+When you start customizing your marketplace with code, you may want to
+continue working on the same web template that powers your hosted
+marketplace, or you may want to develop a fully custom application. Both
+options are possible, and there are some considerations for both
+approaches.
+
+## General custom development guidelines
+
+Whichever custom development approach you choose, here are some key
+points that apply to all custom development.
+
+### Get familiar with the Sharetribe APIs
+
+Your marketplace basic functions are available through the **Marketplace
+API**.
+
+Your no-code configurations are available through the public **Asset
+Delivery API**.
+
+Integration and analytics capabilities are available server-side through
```

Comment:
Can `analytics` be a misleading term here? My first association is Google Analytics or Plausible

Reply:
Fair point, I'll try to find some other way to word this!